userspace/libsinsp/container.cpp

```diff
@@ -207,7 +207,12 @@ bool sinsp_container_manager::set_mesos_task_id(sinsp_container_info* container,
 		if(mtid.empty())
 		{
 			mtid = get_env_mesos_task_id(tinfo);
-			if(!mtid.empty())
+
+			// Ensure that the mesos task id vaguely looks
+			// like a real id. We assume it must be at
+			// least 3 characters and contain a dot or underscore
+			if(!mtid.empty() && mtid.length()>=3 &&
```

Comment:
because of the new test, make the log message on line 222 more accurate and log the task ID found.

Reply:
Ok, updated. I'll squash before merging the PR.